### PR TITLE
Unify path annotations

### PR DESCRIPTION
**Pull request by TonyVlcek**
This unifies the path annotations without changing the internal structure of the hierarchy, all controller methods stay the same.

### CODE REVIEW

File: .docs/endpoints.md [hunk 1]
```diff
@@ -95,17 +94,15 @@ ID
...
-    - Abstract classes only
-    - Defined on controller
-  - `@ControllerPath`
-    - Defined on controller
-  - `@Path`
-    - Defined on method
-  
+  - The `@Path` annotation can be used on:
+    - abstract controller to define a group path for multiple controllers (e.g. `example.com/v1/...`)
+    - final controller to define a path for that particular controller (e.g. `exmaple.com/v1/users`)
```
Comment (mabar) [inline .docs/endpoints.md]: example.com

Comment (TonyVlcek) [reply .docs/endpoints.md]: :+1: Fixed

**Comment by f3l1x** [comment]
👍

**Comment by f3l1x** [comment]
What about GroupId and Id? :-)

**Comment by mabar** [comment]
@f3l1x I was not sure if it's just an annotation change when we talked about paths, so I told him better not to change it.

Looking deeply into it, they can be merged. So if you want to do this change also @TonyVlcek, we would be happy

**Comment by TonyVlcek** [comment]
@mabar @f3l1x Ok, no probs, I'll make a PR for that as well.

**Pull request merged**